packages/server/graphql/mutations/setMeetingSettings.ts

```diff
@@ -0,0 +1,76 @@
+import {GraphQLBoolean, GraphQLID, GraphQLNonNull} from 'graphql'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import getRethink from '../../database/rethinkDriver'
+import {RValue} from '../../database/stricterR'
+import {getUserId} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import standardError from '../../utils/standardError'
+import {GQLContext} from '../graphql'
+import SetMeetingSettingsPayload from '../types/SetMeetingSettingsPayload'
+
+const setMeetingSettings = {
+  type: new GraphQLNonNull(SetMeetingSettingsPayload),
+  description: 'Set default meeting settings',
+  args: {
+    settingsId: {
+      type: new GraphQLNonNull(GraphQLID)
+    },
+    checkinEnabled: {
+      type: GraphQLBoolean,
+      description: 'true to turn icebreaker phase on, false to turn it off'
+    },
+    disableAnonymity: {
+      type: GraphQLBoolean,
+      description: 'disables anonymity of reflections'
+    }
+  },
+  resolve: async (
+    _source: unknown,
+    {
+      settingsId,
+      checkinEnabled,
+      disableAnonymity
+    }: {settingsId: string; checkinEnabled?: boolean; disableAnonymity?: boolean},
+    {authToken, dataLoader, socketId: mutatorId}: GQLContext
+  ) => {
+    const r = await getRethink()
+    const operationId = dataLoader.share()
+    const subOptions = {operationId, mutatorId}
+
+    // AUTH
+    const viewerId = getUserId(authToken)
+    const settings = await r.table('MeetingSettings').get(settingsId).run()
+    if (!settings) {
+      return standardError(new Error('Settings not found'), {userId: viewerId})
+    }
+    const {teamId} = settings
+
+    // RESOLUTION
+    await r
+      .table('MeetingSettings')
+      .get(settingsId)
+      .update((row: RValue) => {
+        const updatedSettings: {[key: string]: any} = {}
+        if (checkinEnabled !== null && checkinEnabled !== undefined) {
```

Comment:
-1 Do we have a Util like `isNil`? Just like Lodash does, to checks if value is null or undefined. [^1] Or we can extract one ourselves.

[^1]: https://lodash.com/docs/4.17.15#isNil